src/Microsoft.DotNet.Interactive.Jupyter/Protocol/LanguageInfo.cs

```diff
@@ -64,14 +64,14 @@ public LanguageInfo(string name, string version, string mimeType, string fileExt
 
     public class CSharpLanguageInfo : LanguageInfo
     {
-        public CSharpLanguageInfo(string version = "8.0") : base("C#", version, "text/x-csharp", ".cs", pygmentsLexer: "csharp")
+        public CSharpLanguageInfo(string version = "9.0") : base("C#", version, "text/x-csharp", ".cs", pygmentsLexer: "csharp")
```

Comment:
If I understand it correctly, C# 9 is supported too

Reply:
Correct, with some caveats that not all C# 9 features may be fully supported in C# scripting.